src/DynamicData/DynamicDataOptions.cs

```diff
@@ -20,4 +21,10 @@ public static class DynamicDataOptions
     /// Gets or sets the system-wide default values for all SortAndBind operations.
     /// </summary>
     public static SortAndBindOptions SortAndBind { get; set; } = new();
+
```

Comment:
Dang.  My original PR had the scheduler on `SortAndBindOption` and I'm now going in circles.

In many respects, it makes sense to have the scheduler on `SortAndBindOption` as all the new binding operators use these whereas I would rather not add it to the original binding options at that would mean I'd have to retrospectively change those after.  

>If you do that way, then you don't need a separate IScheduler parameter in addition to the SortAndBindOptions parameter.
> You can have overloads that take EITHER a SortAndBindOptions OR an IScheduler parameter, and then the latter version can just call the former version with something like DynamicDataOptions.SortAndBind with { Scheduler: scheduler }.

Wise words. 

I'll sit on it for a day or two, but keeping the overloads to a minimum seems like a good idea.

Reply:
> It seems confusing to have ADDITIONAL "Binding" options at the top level when there is already a section for Binding specific options.

It doesn't seem confusing to me. `DynamicDataOptions.BindingScheduler` applies to all "Binding" operators, while `.SortAndBindOptions.Scheduler` applies to just the `.SortAndBind()` operator. I think there's value to be argued for both, but I definitely wouldn't add `DynamicDataOptions.BindingScheduler` without retrofitting it into all the existing operators. That would definitely be counter-intuitive. And I'd say that scope of change should be its own PR.

> If you do that way, then you don't need a separate IScheduler parameter in addition to the SortAndBindOptions parameter.

Agreed. I see no point in having overloads for both `IScheduler` and `SortAndBindOptions`. If there was already an `IScheduler` one, it would make sense to add one for `SortAndBindOptions`, and leave the existing one there to avoid breakage, but that's not the case here.

> In many respects, it makes sense to have the scheduler on SortAndBindOption as all the new binding operators use these

I think this is probably the way to go. That is, having each operator define its own `XXXOptions` struct, instead of just adding optional parameters. It makes adding new options much simpler. And, honestly, I think the point about being forced to retrofit some older operators when adding new options is a BENEFIT. It's a constraint we're placing on ourselves to keep the API a little more sane for consumers. I quite literally think there is only one downside to the `XXXOptions` pattern, and that's the increase in call site size (I.E. every call requires a full allocation of the options struct, regardless of which options are needed). I'm inclined to think this is a cost worth accepting, particularly since operator calls are not hot paths, only the operator innards are.